affectedmoduledetector/src/main/kotlin/com/dropbox/affectedmoduledetector/AffectedModuleDetectorPlugin.kt

```diff
@@ -134,6 +136,9 @@ class AffectedModuleDetectorPlugin : Plugin<Project> {
         val task = rootProject.tasks.register(taskType.commandByImpact).get()
         task.group = groupName
         task.description = taskType.taskDescription
+        if (GradleVersion.current() >= GradleVersion.version("7.4")) {
+            task.notCompatibleWithConfigurationCache("AMD requires knowledge of what has changed in the file system so we can not cache those values (https://github.com/dropbox/AffectedModuleDetector/issues/150)")
```

Comment:
Quite length. Maybe we will devide it to 3 lines?

Reply:
Please do not comment on style issues. We run against ktlint and that is our "code style" so this just adds noise to PRs